### PR TITLE
Prevent retesteth from hanging the CI build

### PR DESCRIPTION
**Pull request by jvdp**
Basically by wrapping it with `timeout 5m`. In addition I bumped the verbosity so it will be easier to see in the logs what's going on.

### CODE REVIEW

File: test-ets.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ final_exit_code=0
 
 function run_and_annotate {
   echo "running retesteth $1"
-  ets/retesteth -t "$1" &> "retesteth-$1-log.txt"
```
Comment (lukasz-golebiewski) [inline test-ets.sh:17]: Could you also `tee` the output to the console so that it's possible to follow what's going on in buildkite?

**Review by lukasz-golebiewski — APPROVED** [review]

**Review by dzajkowski — APPROVED** [review]

File: test-ets.sh [hunk 2]
```diff
@@ -14,7 +14,7 @@ final_exit_code=0
...
+  timeout 5m ets/retesteth -t "$1" -- --verbosity 3 &> "retesteth-$1-log.txt"
```
Comment (leo-bogastry) [inline test-ets.sh]: According to the logs of the build, the ETS tests did run and completed, but the top summary says 
```
retesteth: GeneralStateTests -- --

retesteth crashed; check the artifacts
```
Maybe we need a longer timeout?

**Comment by leo-bogastry** [comment]
It took three runs of the ETS tests, but with the timeout builkite now show the complete final report on the ETS tests

File: test-ets.sh (hunk 1 above)
Comment (jvdp) [reply test-ets.sh:17]: Yes I introduced that in the first commit in this branch actually, to do debugging. But with the timeout the output should eventually end up as artifact. And for developing on ETS it should probably be run locally.

What do you think?

File: test-ets.sh (hunk 2 above)
Comment (jvdp) [reply test-ets.sh]: ETS used to run very quickly (whole suite in a minute or so), I'm kinda curious why it's drastically slower now.

**Pull request merged**